shm/inc/Cargo.hpp

```diff
@@ -2,17 +2,21 @@
 
 #include <string>
 
+constexpr size_t MAX_AMOUNT_OF_CARGO{ 1000 };
+
+//class resposible for manage commodity in game
 class Cargo {
 public:
-    Cargo(const std::string&, size_t, size_t);
+    //constructs an object of cargo in game
+    Cargo(const std::string& name, size_t amount, size_t basePrice);
 
-    Cargo& operator+=(const size_t);
-    Cargo& operator-=(const size_t);
+    Cargo& operator+=(const size_t amount);
+    Cargo& operator-=(const size_t amount);
     bool operator==(const Cargo&) const;
 
     std::string getName() const { return name_; }
     size_t getAmount() const { return amount_; }
-    size_t getBestPrice() const { return basePrice_; }
+    size_t getBasePrice() const { return basePrice_; }
 
 private:
     const std::string name_; 
```

Comment:
Making a const member of the class could be a little tricky. This object can't be easy copy or move constructed :)